assets/css/style.css

```diff
@@ -59,20 +59,58 @@ p {
     background-position: center;
 }
 
+.content {
+    width: 75%;
+    display: inline-block;
+    margin-left: 20px;
+}
+
+
+.search-engine-optimization {
```

Comment:
There is code duplication which can be generalised in to one css selector. Please identify the duplication and reduce the duplication accordingly.